client/my-sites/marketing/connections/service-action.jsx

```diff
@@ -50,6 +50,7 @@ const SharingServiceAction = ( {
 			} );
 		} else {
```

Comment:
Should this also be `warning = true`? If so, maybe we can move it out of the if/else statement.